byte-stream-format-registry-respec.html

```diff
@@ -169,31 +170,31 @@
               audio/webm<br>
               video/webm
             </td>
-            <td><a href="webm-byte-stream-format.html">WebM Byte Stream Format</a></td>
+            <td><a def-id="byte-stream-format-registry-webm"></a> [[!MSE-FORMAT-WEBM]]</td>
```

Comment:
This means a normative reference. I think that's correct, since the MSE spec takes only an informative reference on this registry.